source/faucet/main.d

```diff
@@ -252,7 +252,8 @@ public class Faucet : FaucetAPI
                 this.state.sent_utxos.put(kv.key);
                 return TxBuilder(kv.value.output, kv.key);
             })
-            .map!(txb => txb.unlockSigner(&this.keyUnlocker).split(
+            .map!(txb => txb.unlockSigner(&this.keyUnlocker)
+                .split(
```

Comment:
I don't think this is not related to this commit.

Reply:
I think you mean
`I think this is not related to this commit.`

Reply:
I will split into separate commit.